index.js

```diff
@@ -5,11 +5,11 @@ module.exports = statistics;
 /* Get stats for a file, list of files, or list of messages. */
 function statistics(files) {
   var total = 0;
-  var result = {true: 0, false: 0};
+  var result = {true: 0, false: 0, null: 0};
 
   count(files);
 
-  return {fatal: result.true, nonfatal: result.false, total: total};
+  return {fatal: result.true, nonfatal: result.false, info: result.null, total: total};
```

Comment:
How about adding `'warn'` too? And using nonfatal as all values that are not `fatal: true` (the current behaviour)?